test/test_list.rb

```diff
@@ -768,13 +759,13 @@ def test_nil_position_ordering
     new3.reload.pos = 1
     new3.save
 
-    assert_equal [nil, 1, 2], DefaultScopedMixin.all.map(&:pos)
-    assert_equal [2, 3, 1], DefaultScopedMixin.all.map(&:id)
+    assert_equal [1, 2], DefaultScopedMixin.where("pos IS NOT NULL").map(&:pos)
```

Comment:
Perhaps we should be checking that `new2` has a `nil` position still to keep with the spirit of the original test?